contents/docs/session-replay/how-to-control-which-sessions-you-record.mdx

```diff
@@ -24,10 +34,8 @@ posthog.init('<ph_project_api_key>', {
     disable_session_recording: true,
     // ... other options
 })
-posthog.onFeatureFlags(function () {
-    if (posthog.isFeatureEnabled('your-feature-flag')) {
-        posthog.startSessionRecording()
-    }
+yourCode.eventHandler(function () {
```

Comment:
is `yourCode.eventHandler` the equivalent of a `main` method here? the code snippet isn't clear to me.

Reply:
Hmmm, I was just trying to communicate - "some time later based on a trigger in your system"

I guess it's better to simply say that